frontend/lib/search/results_loader.dart

```diff
@@ -97,7 +96,7 @@ class ResultsLoaderState extends State<ResultsLoader> {
         // Params are outdated.
         // If we're still at the first key, we must manually retrigger fetching.
         if (pageKey == _pagingController.firstPageKey) {
-          return await _fetchPage(pageKey);
+          return _fetchPage(pageKey);
```

Comment:
Do we remove the await because we are returning a Future here?
I like the await syntax more, as it is clearer what is async and what's not 🤔

Reply:
See answer above